### PR TITLE
data store: dont update outputs incrementally

### PR DESCRIPTION
**Pull request by oliver-sanders**
An awkward bodge to work around https://github.com/cylc/cylc-flow/issues/6307

I'm not sure we want to go ahead with this, at least not in the long term, however, it should be enough to unblock (at lest the review of) https://github.com/cylc/cylc-ui/pull/1886

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` (and `conda-environment.yml` if present).
- [x] Tests are included (or explain why tests are not needed).
- [ ] Internal protocol issue, no changelog required
- [x] [Cylc-Doc](https://github.com/cylc/cylc-doc) pull request opened if required at cylc/cylc-doc/pull/XXXX.
- [x] If this is a bug fix, PR should be raised against the relevant `?.?.x` branch.

### CODE REVIEW

**Comment by MetRonnie** [comment]
Looks like b2accfb885b4a8ea7dce1ad2bfdff4d089ac31b1 (`examples: using data to define your workflow`) included accidentally?

**Comment by oliver-sanders** [comment]
(removed)

File: cylc/flow/data_store_mgr.py [hunk 1]
```diff
@@ -2428,11 +2428,25 @@ def delta_task_output(
...
+                _output = tproxy.outputs[_label]
+                output.label = _output.label
+                output.message = _output.message
+                output.satisfied = _output.satisfied
+
+            output.time = update_time
+
         self.updates_pending = True
 
     def delta_task_outputs(self, itask: TaskProxy) -> None:
```
Comment (dwsutherland) [inline cylc/flow/data_store_mgr.py:2452]: Might as well only use this call...

Comment (dwsutherland) [reply cylc/flow/data_store_mgr.py:2452]: Maybe just call `delta_task_outputs` from within `delta_task_output`

Comment (oliver-sanders) [reply cylc/flow/data_store_mgr.py:2452]: This is a bodge workaround, I'm not sure whether we should merge it.

The incremental update is the right thing to do (so I don't want to blast that code aside), we just didn't develop a mechanism for syncing these updates over GraphQL so the UI can't handle it.

**Comment by oliver-sanders** [comment]
Short of a universal solution to #6307, this patch seems a reasonable way to get the info view in the UI off of the ground.

**Review by MetRonnie — APPROVED** [review]

**Pull request merged**